Devscord.DiscordFramework/Framework/Commands/UniversalNumberParser.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Globalization;
+
+namespace Devscord.DiscordFramework.Framework.Commands
+{
+    public class UniversalNumberParser
```

Comment:
jaki jest sens posiadania osobnej klasy
która ma tylko jedną metode
która ma tylko jedną linijke kodu?

Reply:
Ze względu na zasadę "Don't repeat yourself". Ten kod jest używany w dwóch miejscach. Aby w przyszłości nie edytować kodu w dwóch miejscach to wydzieliłem ten kod. Z resztą małe klasy są chyba lepsze niż duże.

PS
Zobacz moje poprzednie odpowiedzi pls.